### DISCUSSION TITLE
Is the value "Approx Entity Count" a real-time value in the latest milvus version, or is it a cumulative value?

### DISCUSSION THREAD

**Discussion by ponponon** [milvus-io/milvus#26635] [Q&A and General discussion]
<img width="1407" alt="图片" src="https://github.com/milvus-io/milvus/assets/38725104/dae82ed3-ff8e-4d1b-a571-a10b43e5fb13">

「Approx Entity Count」这个值，在最新的 milvus 版本中是实时值，还是累加值？之前这个 「Approx Entity Count」 是累加值，即便我删除了向量，这个 「Approx Entity Count」值也不会减小

现在在最新的 milvus 中，这个历史遗留问题已经修复了吗？

-----

Is the "Approx Entity Count" value in the latest milvus version a real time value or an accumulative value? Previously the "Approx Entity Count" was cumulative, and even if I deleted the vector, the "Approx Entity Count" value would not decrease!

Is this historical problem fixed in the latest milvus?

**Comment by xiaofan-luan** [comment]
Approx Entity Count is approximate because it based on meta but not data, so delete is not count.

in milvus 2.3 the count function is accurate but takes resource to run
